BBWP 126: use landscape category

yuki-nmi opened the pull request:
Measure table was updated with new measure scores for water buffering (effect_wb), and landscape-type-specific weighing factor is introduced.
Test functions are updated accordingly.

KeesDoolNMI commented on DESCRIPTION:
```diff
@@ -1,7 +1,7 @@
 Type: Package
 Title: Calculator for BedrijfsBodemWaterPlan (BBWP)
-Version: 2.2.2
+Version: 2.2.3
```
This should be version 2.3.0

yuki-nmi replied:
Indeed! I changed to 2.3.0, also in NEWS.md

KeesDoolNMI reviewed (changes requested):
Nice work, a few final questions

yuki-nmi commented on dev/bbwp_measures2.csv:
now the script to make bbwp_measures table is moved to the Github Repo NMI-DATA_scripts. The products of the script is copied to the folder 'dev'

yuki-nmi reviewed (commented):
Requested changes were made.

yuki-nmi commented:
The syntax of melt function was fixed, so that it is now compatible for data.table 1.16.0.
Further, I found an error in the bbwp_measures.Rd (i.e. a unncessary row-name column was included), so I removed it and updated the test function.

KeesDoolNMI commented:
Fantastic! Thanks for the effort 😄
